neo/app/controllers/Application.scala

```diff
@@ -41,7 +41,7 @@ class Application @Inject()(val wsClient: WSClient,
 
   val manager = system.actorOf(DataStoreManager.props(Migration_20160814.berryMeta, asterixConn, AQLGenerator, config))
 
-  Logger.logger.info("I'm initializing")
+  Logger.info("I'm initializing")
```

Comment:
I also tried this on my local version, `Logger.info` will not print the information on console. `Logger.error` works, `Logger.logger.info` works. Maybe something related to log configuration?